bootstrap/configs.go

```diff
@@ -112,6 +112,10 @@ type ConfigRepository interface {
 	// RemoveChannel removes channel with the given ID.
 	RemoveChannel(ctx context.Context, id string) error
 
+	// ConnectHandler changes state of the Config when the corresponding Thing is
+	// connected to the Channel.
+	ConnectThing(ctx context.Context, channelID, thingID string) error
```

Comment:
Fix comment

Reply:
This has not been fixed. You can rename the function. This applies to `DisconnectHandler`